src/components/shoppable-bar/shoppable-post-widget.js

```diff
@@ -69,8 +69,9 @@ class ShoppablePostWidget {
 
     // Background - poster + blur effect
     const bgSrc = this.player_.cloudinary.currentPoster();
+
     bgSrc.transformation([
-      bgSrc.transformation().toOptions(),
+      bgSrc.transformation().toOptions ? bgSrc.transformation().toOptions() : {},
```

Comment:
```suggestion
      bgSrc.transformation().toOptions?.() ?? {},
```

Reply:
our linter don't like that, failed tests, I tried looking into it but it's a very big linter setup, and it wasn't a simple change, we can add a task to update it